components/Home/HomeDownloadButton.tsx

```diff
@@ -1,41 +1,42 @@
 import LocalizedLink from '../LocalizedLink';
 import { useNextraContext } from '../../hooks/useNextraContext';
 import { getNodejsChangelog } from '../../util/getNodeJsChangelog';
-import type { NodeVersionData } from '../../types';
+import type { NodeReleaseData } from '../../types';
 import type { FC } from 'react';
 
-type HomeDownloadButtonProps = Pick<
-  NodeVersionData,
-  'isLts' | 'node' | 'nodeMajor' | 'nodeNumeric'
->;
+type HomeDownloadButtonProps = {
+  nodeReleaseData: NodeReleaseData;
```

Comment:
```suggestion
  release: NodeReleaseData;
```

Let's simplify this name

Reply:
Also imho, you can make the props of the component itself to be `NodeReleaseData`, this way you don't need to assign to a variable and we can simply destructure the props.